examples/platforms/efr32mg1/README.md

```diff
@@ -0,0 +1,255 @@
+# OpenThread on EFR32MG12 Example
```

Comment:
We should update README for MG1. @mhallam91 may have Silabs help to represent this correctly.

Reply:
@mhallam91 is now @mhallam281